focus-android/app/src/main/java/org/mozilla/focus/activity/MainActivity.kt

```diff
@@ -466,7 +466,11 @@ open class MainActivity : LocaleAwareAppCompatActivity() {
     override fun onDestroy() {
         super.onDestroy()
         _binding = null
-        privateNotificationFeature.stop()
+
+        if (this::privateNotificationFeature.isInitialized) {
+            privateNotificationFeature.stop()
```

Comment:
❓ should we make `privateNotificationFeature` nullable and just use the `?.` syntax?

Reply:
That would need more changes and another approach in initializing it because a `lateinit` property cannot be nullable.